.github/workflows/test.yml

```diff
@@ -6,16 +6,24 @@ name: Run Service Tests
 on:
   push:
     branches: [ dev ]
+    paths-ignore:
+      - 'README.md'
+      - '.github/**'
+      - 'service/local-dev/**'
   pull_request:
     branches: [ '**' ]
+    paths-ignore:
+      - 'README.md'
+      - '.github/**'
+      - 'service/local-dev/**'
   workflow_dispatch:
     inputs:
       testEnv:
         description: 'Environment in which tests should be run. Currently works on dev, alpha and staging'
         required: true
 
 jobs:
-  test:
+  test-job:
```

Comment:
I don't know why the suffix is necessary; this can't be used in any context that doesn't expect a job.